actions/channel_actions.jsx

```diff
@@ -128,6 +128,19 @@ export async function autocompleteChannels(term, success, error) {
     }
 }
 
+export async function autocompleteChannelsByTeamId(teamId, term, success, error) {
```

Comment:
I know you probably did it this way because you're following the pattern of some of the other suggestion providers, but we're actually moving away from these wrappers. Instead, can you just import the `autocompleteChannels` from mattermost-redux directly in the suggestion provider where you need it?

Reply:
Okay fixed! Just curious, how come you're moving away from those action wrappers? They seem more convenient IMO.

Reply:
They're an anti-pattern with React/Redux and all our mattermost-redux actions are async so nothing really gets added by using the wrappers